src/components/ManageUsers/UserSearch.jsx

```diff
@@ -0,0 +1,25 @@
+const UserSearch = () => {
+  return (
+    <main>
+      <div className="usersearch__container">
+        <div>
+          <h1 className="usersearch__h1">Enter user email:</h1>
```

Comment:
it is not a h1. It's a label. See the usage in the login form here: https://github.com/KubaC701/library-webiste/blob/main/src/components/login/Login.jsx#LL38C17-L38C17

Reply:
The "tag" of element  was changed to <label classname="usersearch_label"> ...etc . Class name was changed in each recurrence.